link-hint.el

```diff
@@ -418,6 +419,26 @@ Only search the range between just after the point and BOUND."
   :open-multiple t
   :copy #'kill-new)
 
```

Comment:
Please add a new heading comment before the new type functions and definition (`;; ** Treemacs Link`).